python/lsst/jointcal/jointcal.py

```diff
@@ -189,8 +222,9 @@ def run(self, dataRefs):
             prof = pstats.Stats('jointcal_load_catalog.prof')
             prof.strip_dirs().sort_stats('cumtime').print_stats(20)
         else:
+            old_wcss = []
             for dataRef in dataRefs:
-                self._build_ccdImage(dataRef, associations, jointcalControl)
+                old_wcss.append(self._build_ccdImage(dataRef, associations, jointcalControl))
```

Comment:
Hmm... I added the return of the old WCSes as a convenience for testing (and in case they get altered by future versions of jointcal). We may want to eventually not save the old WCSes, but the list comprehension line isn't too long, so I'll go with it.